### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Ibizugbe**
I have completed this milestone with my project meeting all the requirements of the project.
- I used proper git-flow practice
- The project has no linter error.
- The project is professionally documented
- I have deleted all boilerplate from CRA.
- I have created a folder and named it components.
- I created a `calculator.js` file inside the components folder.
- I have created a react component in `component.js` that matches the specified design.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```
Comment (ShoiraTa) [inline src/index.js]: - [ ] You should try to keep index JS clean, so it should render only `App.js`, and App js file should be the one that imports components like  `Calculator.js`  and renders them. Kindly make sure that the `App.js`(main component) should has the imported `Calculator.js`. App js file should be class based too 😉

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @Ibizugbe ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- 
- Beautiful UI ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (Ibizugbe) [reply src/index.js]: OK, Thanks for the review, I will resolve that now.

**Review by DammyShittu — APPROVED** [review]
Hi @Ibizugbe,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Class components

Great implementation of all the requirements in this milestone. 👏

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**